test/common/tests.go

```diff
@@ -114,6 +114,7 @@ var (
 
 	FAILURE_TESTS = []TestCase{
 		{"C03 - Verify that alerting mechanism works", TestIntegreatlyAlertsMechanism},
+		{"C19 - Validate creation of invalid username triggers alert", TestInvalidUserNameAlert},
```

Comment:
Test is actually IDP related also so could put into that suite to not run this test as part of prow tests (i.e. currently will be run when e2e tests are triggered and will add 5 mins or so to test run time)

Reply:
I think moving it to IDP is a good idea so it doesn't increased the testing time for prow jobs.